.env.sample

```diff
@@ -0,0 +1,3 @@
+// replace <> with database server url to view on server, or 'localhost' to view on port 3000
+
+MONGOHQ_URL=<>
```

Comment:
Could you put a description for the variable in `<>`.

You wouldn't need the description above. It'll then look like:
`MONGOHG_URL=<URL to MondoDB Database>`